packages/maskbook/src/plugins/Ideamarket/UI/Listed.tsx

```diff
@@ -31,9 +34,8 @@ interface ListedProps {
 
 export default function Listed(props: ListedProps) {
     const classes = useStyles()
-    function clicked(e: SyntheticEvent<HTMLButtonElement>) {
-        e.preventDefault()
-        window.open(`https://ideamarket.io/i/twitter/${props.username}`, '_blank', 'noopener')
+    function clicked(e: SyntheticEvent) {
+        e.stopPropagation()
     }
```

Comment:
Actually, this part is also useless now.

Reply:
The tweet is also getting clicked if I don't add that.